### PR TITLE
refactor(combobox)!: rename event

### PR DESCRIPTION
**Pull request by driskull**
BREAKING CHANGE: Renamed event.

- Renamed the event `calciteComboboxChipDismiss`, use `calciteComboboxChipClose` instead.

### CODE REVIEW

File: src/components/combobox/combobox.tsx [hunk 1]
```diff
@@ -308,7 +308,7 @@ export class Combobox
   /**
    * Fires when a selected item in the component is dismissed via its `calcite-chip`.
    */
-  @Event({ cancelable: false }) calciteComboboxChipDismiss: EventEmitter<void>;
```
Comment (benelan) [inline src/components/combobox/combobox.tsx:311]: dismissed => closed in the doc

**Review by benelan — APPROVED** [review]

**Comment by anveshmekala** [comment]
Please attach issue https://github.com/Esri/calcite-components/issues/4688 in the PR description of commit messages for consistency related changes.

**Comment by driskull** [comment]
> Please attach issue https://github.com/Esri/calcite-components/issues/4688 in the PR description of commit messages for consistency related changes.

These are breaking changes. If I put that in the body it will show up in the changelog.

**Comment by anveshmekala** [comment]
> > Please attach issue #4688 in the PR description of commit messages for consistency related changes.
> 
> These are breaking changes. If I put that in the body it will show up in the changelog.

Got you. Didnt knew that. Will attach it [here](https://github.com/Esri/calcite-components/wiki/Consistency-Audit) in wiki.

**Pull request merged**